tutorials/roofit/rf515_hfJSON.py

```diff
@@ -0,0 +1,32 @@
+import ROOT
```

Comment:
```suggestion
## \file
## \ingroup tutorial_roofit
## \notebook
## <Your tutorial title, ending with a dot>
##
## <Little summary of the tutorial, can be multi-line, ending with a dot>
##
## \macro_code
##
## \date November 2021
## \author Carsten Burgard

import ROOT
```

You need to add this header to the tutorial to the ROOT reference guide picks it up.

See for example https://github.com/root-project/root/blob/master/tutorials/roofit/rf612_recoverFromInvalidParameters.py.